addons/sys_radio/initSettings.sqf

```diff
@@ -0,0 +1,19 @@
+private _allRadios = [] call EFUNC(api,getAllRadios);
```

Comment:
I don't think it will be a problem, but I'm not sure if order is guaranteed by this function between clients.
I think it is just based on mod load order, so could be different